.github/workflows/test.yml

```
@@ -51,6 +51,28 @@ jobs:
     - name: Uninstall
       run: sudo make uninstall
 
+  windows:
```

```suggestion
  wine:
```

or something like “cross-windows”